openwisp_controller/connection/signals.py

```diff
@@ -2,5 +2,11 @@
 
 is_working_changed = Signal()
 is_working_changed.__doc__ = """
-Providing araguments: ['is_working', 'old_is_working', 'instance', 'failure_reason']
+Providing araguments: [
+    'is_working',
+    'old_is_working',
+    'instance',
+    'failure_reason',
+    'old_failure_reason'
```

Comment:
This requires an update to the README, but it will be better to do this after https://github.com/openwisp/openwisp-controller/pull/627 is merged and this PR is rebased after.